src/tribler/gui/network/tests/test_request_manager.py

```diff
@@ -8,13 +8,18 @@
 # pylint: disable=protected-access, redefined-outer-name
 
 
+DEFAULT_PORT = 2100
```

Comment:
I think you meant `20100`.

Reply:
I added a `free_port` fixture to GUI tests instead of a constant value, as it looks safer. A similar fixture was already defined in core tests, but not in GUI tests